### ISSUE TITLE
Next wave of onboarding snags

### ISSUE THREAD

**Issue by JonRay15** [vegaprotocol/frontend-monorepo#6949]
### Connection page
  - [x] I dont think it should say "you've been invited to join appName" unless you have a code.... I currently do not and it says it
  - [x] I'm also not sure it should say "Create Wallet" on the button ... it might confuse people?  Maybe just "Connect"

I am also sometimes getting this toast when I land, which makes no sense because the only way you can become invalid is by not meeting the staking requirement, which is zero.  Also once I connect with a key I can indeed apply it.

![Image](https://github.com/user-attachments/assets/ed62abee-8086-4b6b-94d9-4e70b86da8df)

I _think_ there are two things going on here:
1. As per another ticket Console is remembering the last used key in the browser which we dont want
2. I think _that_ key is already in the team which is triggering this very poor error message

So we need to 
- [x] Get that other ticket fixed so Console doesnt remember my key
- [x] Check the error message we get that is triggering this toast

### Deposit form
  - [x] I dont think "Join Nebula" is the right title here ... but never mind for now
  - [x] It should always default to Arbitrum USDT and shouldnt be editable
  - [x] It should always default to your connected key and shouldnt be editable
  - [x] Remove all the "you must deposit 10qUSD etc" text from the top, just put a simple "Min deposit $10" beneath the amount field, maybe only show it if the number is less than that like a standard warning text
  - [x] Change button to DEPOSIT
    - [x] If you need to approve more to process the deposit trigger it 
    - [x] If not just deposit
  - [x] Enhancements to status info
    - [x] Change deposit status to align to normal transaction style in the app
    - [x] Make it so this will finish once some deposit arrives
    - [x] Once it is done it needs to smoothly move you onto the next step with some delay
- [ ] Join Vega page
  - [ ] Need to make sure this covers the case where they are the creator of the team  .... ie teamOwner
  - [ ] The "just want to trade" text is darker than the main text, shoudl be white like the other strapline above
- [x] Use the "Take me to all markets" button that we put on the games page on all the other pages so user can get out at all times
